lnrpc/rpc.proto

```diff
@@ -947,6 +947,13 @@ message ConnectPeerRequest {
     /* If set, the daemon will attempt to persistently connect to the target
      * peer. Otherwise, the call will be synchronous. */
     bool perm = 2;
+
+    /*
+    The connection timeout value for this request. It won't affect other
+    requests. Valid uints are "ms", "s", "m", or "h". For example, to set a
+    timeout value of 30 seconds, use "30s".
+    */
+    string timeout = 3;
```

Comment:
I think it would be more consistent to use `uint64 timeout_seconds` here. Using suffixes is great for cli, but we usually do seconds for rpc.

Reply:
sure, switched.